README.md

```diff
@@ -4,38 +4,34 @@
 
 This is an internal module intended for use by other modules in the [asset-pipe project][asset-pipe].
 
-This module can take one or multiple asset feeds, as produced by the [asset-pipe-js-writer][asset-pipe-js-writer], and produced an executable javascript bundle for the browser.
-
-
+This takes one or more asset feeds, as produced by the [asset-pipe-js-writer][asset-pipe-js-writer], and produces an executable javascript bundle for the browser.
 
 ## Data format
 
-What we refere to as an asset feed is the internal data format used in [Browserify][browserify]. We
-use the exact same data format as Browserify in the [asset-pipe project][asset-pipe].
+What we refer to as asset feeds are arrays of objects in the internal data format used by [Browserify][browserify]. We
+use this exact same data format throughout the [asset-pipe project][asset-pipe].
```

Comment:
"data structure"?

Reply:
Not sure, I think data format is more appropriate... maybe. "Data structure" elicits whiffs of maps along with rich notes of sets and arrays for me.